app/src/main/java/com/jakmos/itemistevolved/presentation/add/AddFragment.kt

```diff
@@ -89,4 +91,11 @@ class AddFragment : BaseFragment(), ItemAdapter.ItemAdapterListener {
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
         menu.clear()
     }
+
+    override fun onDestroy() {
```

Comment:
Nice idea of hiding keyboard.
There is already method for it in viewModel. To extract logic from fragment there should written like that:
`override fun onDestroy() { viewModel.destroy() }`
and destroy should invoke `hideKeyboard()`

Reply:
I see. But I do have to work with `navigation` and `context`. So, your suggetion would be set a `LiveData` inside `fragment` observing this behaviour?

Reply:
Nevermind... Found it. ;)

Reply:
Done